chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
According to the docs (https://docs.chef.io/resource_directory.html) this doesn't work recursively by default. You'd need to add ```recursive true``` here.